features/unit/v2algodclient_paths.feature

```diff
@@ -7,48 +7,58 @@ Feature: Algod REST API v2
     When we make a Shutdown call with timeout <timeout>
     Then expect the path used to be "<path>"
     Examples:
-      |path   | timeout |
-      |TODO   | 0 |
+      |path                   | timeout |
+      |/v2/shutdown           | 0 |
+      |/v2/shutdown?timeout=1 | 1 |
 
   Scenario Outline: Register Participation Keys
     When we make a Register Participation Keys call against account "<account>" fee <fee> dilution <dilution> lastvalidround <lastvalid> and nowait "<nowait>"
     Then expect the path used to be "<path>"
     Examples:
-      |path   | account | fee | dilution | lastvalid | nowait|
-      |TODO   | TODO    | 0   | 0        | 0         | false |
+      |path                                                                                        | account                                                       | fee | dilution | lastvalid | nowait|
+      |/v2/register-participation-keys/7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q  | 7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q    | 0   | 0        | 0         | false |
+      |/v2/register-participation-keys/7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q?fee=1001&key-dilution=23&round-last-valid=12345&no-wait=true  | 7ZUECA7HFLZTXENRV24SHLU4AVPUTMTTDUFUBNBD64C73F3UHRTHAIOF6Q    | 1001   | 23        | 12345         | true |
 
   Scenario Outline: Pending Transaction Information
-    When we make a Pending Transaction Information against txid "<txid>" with max <max>
+    When we make a Pending Transaction Information against txid "<txid>"
     Then expect the path used to be "<path>"
     Examples:
-      |path | txid| max|
-      |TODO | TODO| 0  |
+      |path | txid|
+      |/v2/transactions/pending/5FJDJD5LMZC3EHUYYJNH5I23U4X6H2KXABNDGPIL557ZMJ33GZHQ?format=json | 5FJDJD5LMZC3EHUYYJNH5I23U4X6H2KXABNDGPIL557ZMJ33GZHQ|
+
+  Scenario Outline: Pending Transaction Information
+    When we make a Pending Transaction Information with max <max>
+    Then expect the path used to be "<path>"
+    Examples:
+      |path | max|
+      |/v2/transactions/pending?format=json | 0  |
+      |/v2/transactions/pending?max=1&format=json | 1  |
```

Comment:
@winder is it not `JSON` by default? 
if it is, the better test will be with `msgpack`